src/rp2040.ts

```diff
@@ -1688,27 +1687,21 @@ export class RP2040 {
     else if (opcode === 0b1011111100100000) {
       // do nothing for now. Wait for event!
       this.cycles++;
-      if (this.logMissingInstructions) {
-        console.log('WFE');
-      }
+      this.logger.warn("WFE");
```

Comment:
Those instructions (WFE/WFI/YIELD) should probably be info and not warn, as the code just runs fine without them (at least in the single core variation)

Reply:
Done